.github/workflows/PublishDockerDevImage.yaml

```diff
@@ -0,0 +1,28 @@
+name: Publish Docker dev image
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Build and push Docker image
+        uses: openzim/docker-publish-action@v10
+        with:
+          image-name: openzim/great-project
+          on-master: dev
+          tag-pattern: /^v([0-9.]+)$/
```

Comment:
This allows pushing tags (releases) while this is a `dev` dedicated workflow.
`manual-tag` is more appropriate